include two 5.1.2aplha symbols that were included in XZ 5.1.2alpha which are required by the 'rpm' command on CentOS 7.x
see also https://github.com/easybuilders/easybuild-easyconfigs/issues/4036
original patch: xz-5.2.2-compat-libs.patch, cfr. https://git.centos.org/patch/rpms!xz.git/3f035cee1da9f864609885e5ef2a1be77cd37eee

--- src/liblzma/liblzma.map.orig	2015-09-29 12:57:36.000000000 +0200
+++ src/liblzma/liblzma.map	2017-02-22 11:10:33.432868185 +0100
@@ -95,7 +95,13 @@
 	lzma_vli_size;
 };
 
-XZ_5.2 {
+XZ_5.1.2alpha {
+global:
+   lzma_stream_encoder_mt;
+   lzma_stream_encoder_mt_memusage;
+} XZ_5.0;
+
+XZ_5.2.2 {
 global:
 	lzma_block_uncomp_encode;
 	lzma_cputhreads;
@@ -105,4 +111,4 @@
 
 local:
 	*;
-} XZ_5.0;
+} XZ_5.1.2alpha;
